starknet_py/tests/e2e/utils.py

```diff
@@ -79,3 +80,10 @@ async def get_deploy_account_transaction(
         constructor_calldata=[key_pair.public_key],
         max_fee=MAX_FEE,
     )
+
+
+def _get_random_private_key() -> int:
+    """
+    Returns a private key in the range [1, EC_ORDER).
+    """
+    return random.randint(1, EC_ORDER - 1)
```

Comment:
Maybe lets also remove the docstring? I know I always recommend adding docstring to private methods, but I'd like have a better indication that this is internal and shouldn't be touched. Perhaps lets also have EC_ORDER hardcoded in the function so someone cannot import it from our library?

Reply:
Is it really fair to go as far as removing a function description and hardcoding values into a function, just so a user does not use it themselves? It already is prefixed with `_` which by common convention is private. It also is not a part of our public api.
We could also start defining `__all__` in modules, but it still wouldn't stop users from using our private functions if they decided to.